composer.json

```diff
@@ -25,9 +25,9 @@
         "php": "^7.3",
         "doctrine/common": "^2.7",
         "doctrine/inflector": "^1.1",
-        "knplabs/knp-menu": "^3.1",
-        "knplabs/knp-menu-bundle": "^3.0",
-        "sonata-project/block-bundle": "^4.0",
+        "knplabs/knp-menu": "^2.0 || ^3.1",
+        "knplabs/knp-menu-bundle": "^2.3 || ^3.0",
+        "sonata-project/block-bundle": "^3.18.3 || ^4.0",
```

Comment:
Why these changes ?

Reply:
It is draft. I need it for working demo for now.

Reply:
It is draft. I need it for working demo for now.

Reply:
Ok, sorry for the too early review. 😅